enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/ui/WildflyPluginUtils.java

```diff
@@ -661,6 +667,9 @@ public int compareToIgnoreUpdate(Version o) {
 
         private Version convertEAP(Version version) {
             if (!version.isWidlfy()) {
+                if (version.compareToIgnoreUpdate(EAP_8_0) >= 0) {
+                    return WILDFLY_21_0_0;
```

Comment:
Red Hat thinks JBoss 8 matches Wildfly 28: https://access.redhat.com/solutions/21906

Reply:
Updated to WILDFLY_28_0_0